source/stylesheets/modules/_page_header.scss

```diff
@@ -1,22 +1,32 @@
 @import "bourbon";
+@import "mixins/fonts";
 @import "settings/colors";
 @import "settings/grid_layout";
 
 .page-header {
-  background: $header-background url('/images/header-bg.svg');
+  position: relative;
   color: $header-color;
-  text-shadow: 1px 1px 2px rgba(0,0,0,0.2);
-  @include box-sizing(border-box);
+  overflow: hidden;
+  text-align: center;
 
-  a {
-    color: inherit;
+  .background {
+    min-width: 100%;
```

Comment:
yes it will always be the same width as the parent element, except if the parent element becomes smaller than the original size. This prevents the background from shrinking (and ruining the style) when you make your window narrower.